pkgs/by-name/me/meson/package.nix

```diff
@@ -71,6 +74,8 @@ python3.pkgs.buildPythonApplication rec {
     libxcrypt
   ];
 
+  propagatedNativeBuildInputs = lib.optionals stdenv.isFreeBSD [ buildPackages.freebsd.ldd ];
```

Comment:
Are we bound in freebsd to a single linker? For other platforms this is part of stdenv and can be overwritten.

Reply:
ldd is the dynamic linker, it's used here to print the runtime dependencies of stuff, are you thinking of ld?

Reply:
Ah. Sorry. Mixed them up. Yeah `ldd` is always libc specific by design.